coverage/src/test/java/gov/cms/ab2d/coverage/repository/CoverageServiceRepositoryTest.java

```diff
@@ -105,10 +115,61 @@ void countBeneficiariesByPeriodsWithOptOutTest() {
 
     @Test
     void yearsTest() throws NoSuchFieldException, IllegalAccessException {
-        Field X = CoverageServiceRepository.class.getDeclaredField("YEARS");
-        X.setAccessible(true);
-        Assertions.assertNotNull(X.get(null));
-        Assertions.assertTrue(X.get(null).toString().startsWith("[2020, 2021, 2022, 2023, 2024"));
+        Field field = CoverageServiceRepository.class.getDeclaredField("YEARS");
+        field.setAccessible(true);
+        Assertions.assertNotNull(field.get(null));
+        Assertions.assertTrue(field.get(null).toString().startsWith("[2020, 2021, 2022, 2023, 2024"));
```

Comment:
Only comment is if you would consider extending this to future years too?

Reply:
If you look at the diff closely, you'll notice that the only thing I changed in this test was changing a variable from `X` to `field`. I did that to resolve a sonarqube code smell. I didn't write this test, so I'm not sure what it's doing or if it needs to be extended in any way.

Reply:
Understood. Something that need to be considered for later may be then!Thank you for notifying me :)